internal/pkg/cli/pipeline_status.go

```diff
@@ -47,12 +61,12 @@ func BuildPipelineStatusCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Hidden: true, // TODO: remove when ready for production!
 		Use:    "status",
-		Short:  "Show the status of your application's pipeline.",
+		Short:  "Show the status of a pipeline.",
 		Long:   "Show the status of each deployed pipeline's stages.",
 
 		Example: `
-Shows status of the deployed pipeline "pipeline-myapp-mycompany-myrepo".
-/code $ copilot pipeline status -n pipeline-myapp-mycompany-myrepo`,
+Shows status of the pipeline "pipeline-myapp-myrepo".
+/code $ copilot pipeline status -n pipeline-myapp-myrepo`,
```

Comment:
any reason why we're taking out "mycompany"?

Reply:
@kohidave

Reply:
just to keep the suggested lines a little short. i don't have any great reasoning behind it other than i just thought it looked less cluttered